fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/PredicatesPushDown.java

```diff
@@ -0,0 +1,9 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.optimizer.operator;
+
+public interface PredicatesPushDown {
```

Comment:
I tend to remove this and add a base class LogicalHiveOperator and LogicalHudiOperator. Since this class name is a little misleading, we have many transformers and rule implementations for PushDown.